ebuild-maintenance/general/text.xml

```diff
@@ -0,0 +1,527 @@
+and replicate it with the information attached to the bug. You should then
+either mark the bug as "CONFIRMED", or provide a comment that you can't.
+</p>
+
+<p>
+Keep an eye out for CI/tinderbox-style bugs too, which are found by testing the
+latest ebuilds (by commit) and random packages, respectively. They reveal
+breakages on different systems, mostly missing dependencies, and make you face
+issues you perhaps hadn't thought of before.
+</p>
+
+<p>
+When you make a commit to Gentoo's git tree, you can reference or close a
+single bug, or multiple bugs, by using <e>Bug:</e> or <e>Closes:</e> tag in
+your git commit message's footer area.
+<uri link="::ebuild-maintenance/git/#Git commit message format">See an example,
+and more documentation</uri>.
+</p>
+
+<p>
+<uri link="::keywording/#Stabilization rules">Stabilization can be
+done by a maintainer who is also a developer, and owns relevant hardware</uri>.
+It can also be <uri link="::keywording/#Moving from ~arch to arch">
+requested through Bugzilla</uri> for arch teams to handle, even for your own
+packages.
+</p>
+
+<p>
+It is recommended to set up your mail client's filtering properly. Separating
+Bugzilla mail from generic alias mail, mailing lists, GitHub etc, helps to
+organize your workflow and time usage.
+</p>
+
+<p>
+A simple alias separation, for example, is done by simple filters:
+</p>
+
+<pre>
+# example project, Bugzilla mail
+To-contains: example@gentoo.org
+From-is: bugzilla-daemon@gentoo.org
+
+# example project, alias mail
+To-contains: example@gentoo.org
+From-isn't: bugzilla-daemon@gentoo.org
+</pre>
+
+<p>
+And separate your personal Bugzilla mail such as:
+</p>
+
+<pre>
+# Personal Bugzilla mail
+To-is: developer@gentoo.org
+From-is: bugzilla-daemon@gentoo.org
+</pre>
+
+<p>
+You can separate mailing list e-mails similarly, etc.
+</p>
+
+<p>
+Any GitHub pull requests linked to your bugs are shown in the "See also" field.
+See the <uri link="::ebuild-maintenance/general/#GitHub pull requests"/>
+section below.
+</p>
+
+<p>
+Check bugs assigned to you by searching <c>assignee:developer@gentoo.org</c>.
+You can also see them in <uri link="https://bugs.gentoo.org/">Bugzilla's front
+page</uri> after logging in, "Open bugs assigned to me". And also in
+<uri link="::ebuild-maintenance/general/#Package updates, and bugs">
+packages.gentoo.org's maintainer page </uri>.
+</p>
+
+</body>
+</subsection>
+
+<subsection>
+<title>Ebuilds &amp; git workflow</title>
+<body>
+
+<p>
+See <uri link="::ebuild-maintenance/git">Git for Gentoo Developers</uri> for
+configuration help.
+</p>
+
+<p>
+When writing and pushing ebuilds to Gentoo's repository tree (and why not to
+any overlay as well) you should definitely take advantage of available QA
+tools. <c>pkgcheck</c> and <c>repoman</c> being your choices here. They can
+both be found and installed from the main Gentoo repository with their
+respective names.
+</p>
+
+<!-- author's note to reviewers: I find the PMS link correct here, but
```

Comment:
I'd link to the identifier from a specific EAPI just in case, so use 7 from now.

Reply:
That did cross my mind but I thought if at some point more text was added to it, clarifications, updates etc, they don't make it to older versions. But this is a good idea, I'll do that.